etc/picongpu/hemera-hzdr/k80.tpl

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+# Copyright 2013-2019 Axel Huebl, Richard Pausch, Rene Widera, Marco Garten
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+# PIConGPU batch script for hemera' SLURM batch system
+
+#SBATCH --partition=!TBG_queue
+# necessary to set the account also to the queue name because otherwise access is not allowed at the moment
+#SBATCH --account=!TBG_queue
+#SBATCH --time=!TBG_wallTime
+# Sets batch job's name
+#SBATCH --job-name=!TBG_jobName
+#SBATCH --nodes=!TBG_nodes
+#SBATCH --ntasks=!TBG_tasks
+#SBATCH --ntasks-per-node=!TBG_gpusPerNode
+#SBATCH --mincpus=!TBG_mpiTasksPerNode
+#SBATCH --cpus-per-task=!TBG_coresPerGPU
+#SBATCH --mem=!TBG_memPerNode
+#SBATCH --gres=gpu:!TBG_gpusPerNode
+#SBATCH --mail-type=!TBG_mailSettings
+#SBATCH --mail-user=!TBG_mailAddress
+#SBATCH --workdir=!TBG_dstPath
+#SBATCH --workdir=!TBG_dstPath
+
+#SBATCH -o stdout
+#SBATCH -e stderr
+
+
+## calculations will be performed by tbg ##
+.TBG_queue="k80"
+
+# settings that can be controlled by environment variables before submit
+.TBG_mailSettings=${MY_MAILNOTIFY:-"NONE"}
+.TBG_mailAddress=${MY_MAIL:-"someone@example.com"}
+.TBG_author=${MY_NAME:+--author \"${MY_NAME}\"}
+.TBG_profile=${PIC_PROFILE:-"~/picongpu.profile"}
+
+# number of available/hosted GPUs per node in the system
+.TBG_numHostedGPUPerNode=8
+
+# required GPUs per node for the current job
+.TBG_gpusPerNode=`if [ $TBG_tasks -gt $TBG_numHostedGPUPerNode ] ; then echo $TBG_numHostedGPUPerNode; else echo $TBG_tasks; fi`
+
+# host memory per gpu
+.TBG_memPerGPU="$((220000 / $TBG_gpusPerNode))"
```

Comment:
So `sinfo -lp k80 -N` shows the following

```Shell
garten70@hemera5:~$ sinfo -lp k80 -N
Tue Mar 12 16:41:38 2019
NODELIST   NODES PARTITION       STATE CPUS    S:C:T MEMORY TMP_DISK WEIGHT AVAIL_FE REASON              
kepler020      1       k80        idle   16    2:8:1 240000        0      1   (null) none                
kepler021      1       k80        idle   16    2:8:1 240000        0      1   (null) none                
kepler022      1       k80        idle   16    2:8:1 240000        0      1   (null) none                
kepler023      1       k80        idle   16    2:8:1 240000        0      1   (null) none                
kepler024      1       k80       down*   16    2:8:1 240000        0      1   (null) Not responding      
kepler025      1       k80        idle   16    2:8:1 240000        0      1   (null) none                
kepler026      1       k80        idle   16    2:8:1 240000        0      1   (null) none                
kepler027      1       k80        idle   16    2:8:1 240000        0      1   (null) none                
kepler028      1       k80        idle   16    2:8:1 240000        0      1   (null) none 
```

When I did that for the `gpu` queue I saw that the host memory is `380000`GB but we leave `20000` free and just use `360000`. So I did the same for the `k80` nodes.
Is that fine or should we do it differently?

Reply:
It's smart to use slightly less than the full node, even if the batch system does not enforce it.
Leaving maybe 1-2 GByte instead of 20 free is what I would suggest ;-)

Reply:
Then we can do the same for the P100 nodes and reduce the free space from 20GB to 1-2GB as well, right?

Reply:
I would think so, if @psychocoderHPC doesn't object.

Reply:
yes it should be more like 2GiB instead of 20GiB